include/jemalloc/internal/mutex_pool_structs.h

```diff
@@ -0,0 +1,14 @@
+#ifndef JEMALLOC_INTERNAL_MUTEX_POOL_STRUCTS_H
+#define JEMALLOC_INTERNAL_MUTEX_POOL_STRUCTS_H
+
+/* This file really combines "structs" and "types", but only transitionally. */
+
+/* We do mod reductions by this value, so it should be kept a power of 2. */
+#define MUTEX_POOL_SIZE 256
```

Comment:
I figured that for something at the same level of the stack as a mutex, having it not allocate (or at least, not allocate any more than a malloc_mutex_t) would be useful, so I just picked a number sufficiently large that I assumed no real contention ever. Are you thinking to just get the memory from ialloc?